server/src/main/java/com/coincoinche/ratingplayer/EloService.java

```diff
@@ -25,19 +27,18 @@ public EloService() {
    *
    * @param winnerTeam is the coinche team winning the game.
    * @param loserTeam is the coinche team losing the game.
+   * @return a Map containing the new elo rating per username.
    */
-  public void updateRatings(Team winnerTeam, Team loserTeam) {
-    try {
-      EloTeam winner = eloTeamFactory.createEloTeamFromCoincheTeam(winnerTeam);
-      EloTeam loser = eloTeamFactory.createEloTeamFromCoincheTeam(loserTeam);
-      updateEloTeamRatings(winner, loser);
-      logger.debug("Successfully updated Elo ratings");
-    } catch (Exception e) {
-      logger.error(String.format("{}: skip Elo ratings update", e));
-    }
+  public Map<String, Integer> updateRatings(Team winnerTeam, Team loserTeam)
+      throws UserNotFoundException {
+    EloTeam winner = eloTeamFactory.createEloTeamFromCoincheTeam(winnerTeam);
+    EloTeam loser = eloTeamFactory.createEloTeamFromCoincheTeam(loserTeam);
+    logger.debug("Successfully updated Elo ratings");
```

Comment:
Can we log this after the rating is actually updated? Or we could also just remove this log.